JavaScript/JavaScriptSDK.Tests/CheckinTests/appInsights.tests.ts

```diff
@@ -1796,8 +1792,33 @@ class AppInsightsTests extends TestClass {
                 (<any>xhr).respond("200", {}, "");
 
                 // Assert
-                Assert.equal(null, (<any>xhr).requestHeaders['x-ms-request-id'], "x-ms-request-id should not be set");
```

Comment:
Do we need any backward compatibility and still support `x-ms'*`  headers? Breeze will understand `Request-ID`, but how about Vortex?

Reply:
didn't understand this... this format is for all server-side SDKs, but not for Breeze or Vortex... how those are involved here?